### PR TITLE
Ana/295 incorrect ledger error

### LINKED ISSUE #295: this ledger error is incorrect
this was in brave after cancelling the little device popup screen

we can probably just remove it and let the `error` field take care of it

Screen Shot 2020-11-26 at 12 09 27 PM

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #295 

Tested with Firefox and it works

image

### CODE REVIEW

File: pages/ledger/index.vue [hunk 1]
```diff
@@ -58,7 +58,7 @@
       <br />
       <br />
     </div>
-    <div v-if="!isWindows && !isLinux" class="session-main">
+    <div v-if="!isChromiumBrowser" class="session-main">
```
Comment (jbibla) [inline pages/ledger/index.vue:61]: ok cool - i see the fix. what i was trying to say is that if this is an error message, why not just show it in {{ error }}? do we need a special message just for this one case?

Comment (jbibla) [reply pages/ledger/index.vue:61]: maybe we make this a computed property where it sets the `error` if it's a chromium browser

Comment (Bitcoinera) [reply pages/ledger/index.vue:61]: I think there are too many possible errors, Jordan. Depends if it is IE, Opera, Firefox... there are just too many browsers out there. This is better, trust me.

In Firefox the error is "there is no `window.usb`", in Safari is another... Too many possibilities

Comment (jbibla) [reply pages/ledger/index.vue]: nevermind. all good.

**Pull request merged**